boards/arm/stm32/stm32f4discovery/scripts/Make.defs

```diff
@@ -35,7 +35,11 @@ ifneq ($(CONFIG_DEBUG_NOOPT),y)
 endif
 
 ARCHCFLAGS = -fno-builtin
-ARCHCXXFLAGS = -fno-builtin -fno-exceptions -fcheck-new
+ifeq ($(CONFIG_CXX_EXCEPTION),y)
+  ARCHCXXFLAGS = -fno-builtin
+else
+  ARCHCXXFLAGS = -fno-builtin -fno-exceptions -fcheck-new
```

Comment:
Why we do not have `-fno-rtti` here?

Reply:
Add at line 52:
https://github.com/apache/incubator-nuttx/pull/5555/files#diff-0a874401241f632c07d97d166345317a75e439db54eb33b3082394ba2a6749bbL48